geode-core/src/main/resources/META-INF/schemas/geode.apache.org/schema/cache/cache-1.0.xsd

```diff
@@ -1191,6 +1191,7 @@ As of 6.5 disk-dirs is deprecated on region-attributes. Use disk-store-name inst
         </xsd:complexType>
       </xsd:element>
     </xsd:choice>
+    <xsd:attribute name="socket-connect-timeout" type="xsd:string" use="optional" />
```

Comment:
I've opened a discussion on geode-dev about this change.  I would think a new XSD needs to be created.